### PR TITLE
possible improve of the contract 

### PR DESCRIPTION
**Pull request by tina1998612**

### CODE REVIEW

File: contracts/Asset.sol [hunk 1]
```diff
@@ -4,11 +4,11 @@ contract Asset {
     
     // ######### Parameters ######## //
     
-    string public name;
     address public publisher;
     address public holder;
-    string public status;
```
Comment (tina1998612) [inline contracts/Asset.sol]: can combine "status" and "avail"

File: contracts/Asset.sol [hunk 2]
```diff
@@ -29,64 +29,48 @@ contract Asset {
         _;
     } 
     
-    modifier onlyHolderOrPublisher(address sender){
```
Comment (tina1998612) [inline contracts/Asset.sol:32]: this is not used in this contract (?

File: contracts/Asset.sol [hunk 3]
```diff
@@ -29,64 +29,48 @@ contract Asset {
...
-        require(sender == holder || sender == publisher);
-        _;
-    } 
     
-    modifier isAvail(){
-        require(avail == true);
+    modifier onlyAvail(){
```
Comment (tina1998612) [inline contracts/Asset.sol]: naming consistency

**Comment by yhuag** [comment]
Tested and looks okay I guess

**Pull request merged**